specification/search/data-plane/Azure.Search/preview/2024-05-01-Preview/searchservice.json

```
@@ -0,0 +1,12432 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "SearchServiceClient",
+    "description": "Client that can be used to manage and query indexes and documents, as well as manage other resources, on a search service.",
+    "version": "2024-05-01-Preview",
```

And please also change this one:
```suggestion
    "version": "2024-05-01-preview",
```

---

addressed it